app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -8,14 +8,15 @@ import { Header, SubHeader } from "neetoui/layouts";
 import NoteTable from "./NoteTable";
 import NewNotePane from "./NewNotePane";
 import DeleteAlert from "./DeleteAlert";
+import { NotesProvider, useNotes } from "contexts/notes";
```

Comment:
Please sort import statements. 
Please refer [Sorting import statements](https://bigbinary.com/react-best-practices/sort-import-statements#sorting-import-statements) for more details.